app/views/hub/efile_submissions/show.html.erb

```diff
@@ -1,22 +1,28 @@
+<% content_for :back_to, "efile" %>
 <% content_for :card do %>
-  <div class="slab slab--not-padded spacing-above-25">
-    <h1 class="h2 spacing-below-5">
-      Submission <%= @efile_submission.id %> <span class="small">(<%= @efile_submission.current_state %>)</span>
-    </h1>
-    <p class="spacing-above-5 spacing-below-5">Client: <%= link_to @efile_submission.tax_return.client.legal_name, hub_clients_path(id: @efile_submission.tax_return.client.id) %></p>
+  <div class="slab slab--not-padded spacing-above-25 submission">
+    <div style="display: flex;">
+      <h1 class="h2 spacing-below-5">
+        <%= link_to @tax_return.client.legal_name, hub_clients_path(id: @tax_return.client.id) %> (<%= @efile_submission.current_state %>)
+      </h1>
+      <div style="margin-left: 10px;">
+        <%= link_to "Resubmit return", resubmit_hub_efile_submission_path(id: @efile_submission.id), method: :patch, data: { confirm: "Have you updated the necessary information for #{@tax_return.client.legal_name}?" },  class: "button button--small" if @efile_submission.can_transition_to?(:resubmitted) %>
```

Comment:
✨

Reply:
If the submission can't be transitioned to resubmitted, presumably the reason is that it's already been resubmitted. If so, it makes sense to me we'd provide a link to the next submission's EfileSubmission page.

Not a blocker, but curious if you agree.

Reply:
all of the submissions are on this page. thats why it displays on a tax return basis.